.github/workflows/main.yml

```diff
@@ -163,7 +163,7 @@ jobs:
           overwrite: true
 
   test-build:
-    name: verify packages / python ${{ matrix.python-version }} / ${{ matrix.os }}
+    name: verify packages / python ${{ matrix.python-version }} / ${{ matrix.os }} / ${{ matrix.dist-type }} 
```

Comment:
I think you're getting dinged for the extra space at the end of this line. Try running `precommit`.

Reply:
I promise I'll run this